Source/WarpX.cpp

```diff
@@ -853,6 +853,13 @@ WarpX::ReadParameters ()
         pp_warpx.query("do_pml_j_damping", do_pml_j_damping);
         pp_warpx.query("do_pml_in_domain", do_pml_in_domain);
         pp_warpx.query("do_similar_dm_pml", do_similar_dm_pml);
+        // Read `v_particle_pml` in units of the speed of light
+        v_particle_pml = 1;
```

Comment:
Just adding `_rt` for consistency:
```suggestion
        v_particle_pml = 1._rt;
```